### ISSUE TITLE
View y origin issue

### ISSUE THREAD

**Issue by Hamza9512** [hackiftekhar/IQKeyboardManager#1551]
I try to change view.origin.y position when textFiled tapped and keyboard revealed. View changing y position and going up but immediately going down again. When i disable IQKeyboardManager from AppDelegate it's working. Video attached

https://www.dropbox.com/s/6ol6g1lzjdjstgv/Screen%20Recording%202019-05-08%20at%2016.43.59.mov?dl=0

**Comment by M-Quadra** [comment]
I guess it would be helpful

```
if (@available(iOS 11.0, *)) {
    [[UIScrollView appearance] setContentInsetAdjustmentBehavior:UIScrollViewContentInsetAdjustmentNever];
}
```

**Comment by hackiftekhar** [comment]
@Hamza9512 This is because library task is to change ViewController's view.origin.y position to make textField visible. If you are doing the same thing then you are doing is wrong. Either library should do it or you should with your own implementation.

When you disable library then it works because you tell library to not do anything and you are performing view origin change.
